package.json

```diff
@@ -152,8 +148,5 @@
         "cypress:ci": "cross-env BABEL_COVERAGE=1 cypress run --component --env CYPRESS_COVERAGE=1",
         "format": "prettier --write .",
         "typecheck": "tsc"
-    },
-    "dependencies": {
-        "tiny-invariant": "^1.3.1"
```

Comment:
Now that we only have `dependencies`, this has moved up into the main body of dependencies above.